### ISSUE TITLE
[Real Device iPhone] Safari repeatedly loads about:blank page instead of test page when testing with Intern test framework

### ISSUE THREAD

**Issue by skbeacon** [appium/appium#6183]
When trying to do my test on a webpage on Safari of iPhone, it loops for 18 times loading about:blank page instead of loading my intended webpage. It throws error which is given below.

**Environment**
* Appium version: 1.4.8
* Mac OSX: 10.9.5
* Node.js version (unless using Appium.app|exe):
* iOS 7.0.3 (iPhone 5 - Real device)
* Test framework: intern (https://theintern.github.io/intern/#what-is-intern)
* ios-webkit-debug-proxy version: 1.6

**Please Note:**
* I am trying to test a web app on Safari
* ios-webkit-debug-proxy is running
* Web inspector option of Safari is enabled.
* All neccessary libs are installed.
* **This same test is running smoothly in Android.**

**Intern Configuration for iOS**
environments: [		
		{
			browserName: 'Safari',
			platformName: 'iOS',
                        platformVersion: '7.0.3',
	                app: 'io.appium.SafariLauncher',
                        deviceName: <UDID>,
		        udid: <UDID>,
                        safariIgnoreFraudWarning: true,
                        safariAllowPopups: false,
                        automationName: 'Appium'
		}
	],

Can anyone help me out with this? Its very important for me to complete the tests and I am stuck for a week with this. I tried different solutions I found in the sites but nothing could help me to solve my problem. I am giving the error log below:

info: --> POST /wd/hub/session {"desiredCapabilities":{"browserName":"Safari","platformName":"iOS","platformVersion":"7.0.3","app":"io.appium.SafariLauncher","deviceName":"6ea762d97e84b78f78ba6865","udid":"6ea762d97e84b78f78ba67e865","automationName":"Appium","name":"./intern_test_codes/ios.mweb.config.js","idle-timeout":60,"mouseEnabled":true,"nativeEvents":true,"locationContextEnabled":true}}
info: Client User-Agent string: dojo/2.0.0-dev Node.js/5.0.0
info: [debug] The following desired capabilities were provided, but not recognized by appium. They will be passed on to any other services running on this server. : name, idle-timeout, mouseEnabled, nativeEvents, locationContextEnabled
info: [debug] Configuring Safari session
info: [debug] Using local .zip from desired caps: **<MY PROJECT DIR>**/appium/build/SafariLauncher/SafariLauncher.zip
info: [debug] Copying local zip to tmp dir
info: [debug] **<MY PROJECT DIR>**/appium/build/SafariLauncher/SafariLauncher.zip copied to /var/folders/kr/hjwhz7k9301d0bj6_7rb4lch0000gn/T/116124-2357-1immlcm/appium-app.zip
info: [debug] Unzipping /var/folders/kr/hjwhz7k9301d0bj6_7rb4lch0000gn/T/116124-2357-1immlcm/appium-app.zip
info: [debug] Testing zip archive: /var/folders/kr/hjwhz7k9301d0bj6_7rb4lch0000gn/T/116124-2357-1immlcm/appium-app.zip
info: [debug] Zip archive tested clean
info: [debug] Unzip successful
info: [debug] Using locally extracted app: /var/folders/kr/hjwhz7k9301d0bj6_7rb4lch0000gn/T/116124-2357-1immlcm/SafariLauncher.app
info: [debug] Creating new appium session a094ed38-adc2-4e18-a34d-946c06b53d59
info: [debug] Removing any remaining instruments sockets
info: [debug] Cleaned up instruments socket /tmp/instruments_sock
info: [debug] Auto-detecting iOS udid...
info: [debug] Not auto-detecting udid, running on sim
info: [debug] Could not parse plist file (as binary) at /var/folders/kr/hjwhz7k9301d0bj6_7rb4lch0000gn/T/116124-2357-1immlcm/SafariLauncher.app/en.lproj/Localizable.strings
info: Will try to parse the plist file as XML
info: [debug] Could not parse plist file (as XML) at /var/folders/kr/hjwhz7k9301d0bj6_7rb4lch0000gn/T/116124-2357-1immlcm/SafariLauncher.app/en.lproj/Localizable.strings
warn: Could not parse app Localizable.strings assuming it doesn't exist
info: [debug] Getting bundle ID from app
info: [debug] Parsed app Info.plist (as binary)
info: [debug] Creating instruments
info: [debug] Preparing uiauto bootstrap
info: [debug] Dynamic bootstrap dir: /Users/<user>/Library/Application Support/appium/bootstrap
info: [debug] Dynamic env: {"nodePath":"/usr/local/bin/node","commandProxyClientPath":"**<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js","instrumentsSock":"/tmp/instruments_sock","interKeyDelay":null,"justLoopInfinitely":false,"autoAcceptAlerts":false,"autoDismissAlerts":false,"sendKeyStrategy":"grouped"}
...o: [debug] Dynamic bootstrap code: // This file is automatically generated. Do not manually modify!
info: [debug] Dynamic bootstrap path: /Users/<user>/Library/Application Support/appium/bootstrap/bootstrap-70591bc7ca981e08.js
info: [debug] Reusing dynamic bootstrap: /Users/<user>/Library/Application Support/appium/bootstrap/bootstrap-70591bc7ca981e08.js
info: [debug] Attempting iOS device log capture via libimobiledevice idevicesyslog
info: [debug] Creating iDevice object with udid 6ea762d97e84b78f78ba67e2e8c8826ca5f8f865
info: [debug] Starting command proxy.
info: [debug] Instruments socket server started at /tmp/instruments_sock
info: [debug] Starting instruments
info: [debug] Instruments is at: /Applications/Xcode_6.1.app/Contents/Developer/usr/bin/instruments
info: Launching instruments
info: [debug] Attempting to run app on real device with UDID 6ea762d97e84b78f78ba67e2e8c8826ca5f8f865
info: [debug] Spawning instruments with command: /Applications/Xcode_6.1.app/Contents/Developer/usr/bin/instruments -t /Applications/Xcode_6.1.app/Contents/Applications/Instruments.app/Contents/PlugIns/AutomationInstrument.xrplugin/Contents/Resources/Automation.tracetemplate -D /tmp/appium-instruments/instrumentscli0.trace -w 6ea762d97e84b78f78ba67e2e8c8826ca5f8f865 com.widespace.sarilauncher -e UIASCRIPT "/Users/<user>/Library/Application Support/appium/bootstrap/bootstrap-70591bc7ca981e08.js" -e UIARESULTSPATH /tmp/appium-instruments
info: [debug] And extra without-delay env: {}
info: [debug] And launch timeouts (in ms): {"global":90000}
info: [debug] [INST STDERR] 2016-02-24 13:21:47.124 instruments[2370:5803] WebKit Threading Violation - initial use of WebKit from a secondary thread.
info: Instruments is ready to receive commands
info: [debug] Instruments launched. Starting poll loop for new commands.
info: [debug] Setting bootstrap config keys/values
info: [debug] Pushing command to appium work queue: "target = $.target();\nau = $;\n$.isVerbose = true;\n"
info: [debug] Socket data received (2 bytes)
info: [debug] Socket data being routed.
info: [debug] Sending command to instruments: target = $.target();
au = $;
$.isVerbose = true;

info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫১ +0000 Debug: evaluation finished
       ২০১৬-০২-২৪ ০৭:২১:৫১ +0000 Debug: responding with:
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫১ +0000 Debug: Running system command #1: /usr/local/bin/node **<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":true}...
info: [debug] Socket data received (27 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":true}
info: [debug] Not setting initial orientation because we're on SafariLauncher
info: [debug] Waiting for app source to contain elements
info: [debug] Pushing command to appium work queue: "au.mainApp().getTreeForXML()"
info: [debug] Sending command to instruments: au.mainApp().getTreeForXML()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫২ +0000 Debug: Got new command 1 from instruments: au.mainApp().getTreeForXML()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫২ +0000 Debug: evaluating au.mainApp().getTreeForXML()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫২ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫২ +0000 Debug: responding with:bled":true,"valid":true,"visible":true,"hint":null,"path":"/0/0","x":0,"y":0,"width":320,"height":568},">":[{"UIAStaticText":{"@":{"name":"Safari Launcher","label":"Safari Launcher","value":"Safari Launcher","dom":null,"enabled":true,"valid":true,"visible":true,"hint":null,"path":"/0/0/0","x":80,"y":10,"width":320,"height":50},">":[]}},{"UIAButton":{"@":{"name":"launch safari","label":"launch safari","value":null,"dom":null,"enabled":true,"valid":true,"visible":true,"hint":null,"path":"/0/0/1","x":60,"y":254,"width":200,"height":60},">":[]}},{"UIAStaticText":{"@":{"name":"   url: http://www.apple.com","label":"   url: http://www.apple.com","value":"   url: http://www.apple.com","dom":null,"enabled":true,"valid":true,"visible":true,"hint":null,"path":"/0/0/2","x":0,"y":518,"width":320,"height":50},">":[]}}]}},{"UIAWindow":{"@":{"name":null,"label":null,"value":null,"dom":null,"enabled":true,"valid":true,"visible":true,"hint":null,"path":"/0/1","x":0,"y":0,"width":320,"height":568},">":[{"UIAStatusBar":{"@":{"name":null,"label":null,"value":null,"dom":null,"enabled":true,"valid":true,"visible":true,"hint":null,"path":"/0/1/0","x":0,"y":0,"width":320,"height":20},">":[{"UIAElement":{"@":{"name":"No SIM","label":"No SIM","value":null,"dom":null,"enabled":true,"valid":true,"visible":true,"hint":"Swipe down with three fingers to reveal the notification center., Swipe up with three fingers to reveal the control center, Double-tap to scroll to top","path":"/0/1/0/0","x":6,"y":0,"width":42,"height":20},">":[]}},{"UIAElement":{"@":{"name":"3 of 3 Wi-Fi bars","label":"3 of 3 Wi-Fi bars","value":"SQA Airport Express","dom":null,"enabled":true,"valid":true,"visible":true,"hint":"Swipe down with three fingers to reveal the notification center., Swipe up with three fingers to reveal the control center, Double-tap to scroll to top","path":"/0/1/0/1","x":53,"y":0,"width":13,"height":20},">":[]}},{"UIAElement":{"@":{"name":"13:21","label":"13:21","value":null,"dom":null,"enabled":true,"valid":true,"visible":true,"hint":"Swipe down with three fingers to reveal the notification center., Swipe up with three fingers to reveal the control center, Double-tap to scroll to top","path":"/0/1/0/2","x":146,"y":0,"width":28,"height":20},">":[]}},{"UIAElement":{"@":{"name":"100% battery power, On AC Power","label":"100% battery power, On AC Power","value":null,"dom":null,"enabled":true,"valid":true,"visible":true,"hint":"Swipe down with three fingers to reveal the notification center., Swipe up with three fingers to reveal the control center, Double-tap to scroll to top","path":"/0/1/0/3","x":248,"y":0,"width":67,"height":20},">":[]}}]}}]}}]}}}
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫২ +0000 Debug: Running system command #2: /usr/local/bin/node **<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":{"UIAApplication":{"@":{"name":"SafariLauncher","label":"SafariLauncher","va...
info: [debug] Socket data received (2963 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":{"UIAApplication":{"@":{"name":"SafariLauncher","label":"SafariLauncher","value":null,"dom":null,"enabled":true,"valid":true,"visible":true,"hint":null,"path":"/0","x":0,"y":20,"width":320,"height":548},">":[{"UIAWindow":{"@":{"name":null,"label":null,"value":null,"dom":null,"ena
info: [debug] On Safari Launcher. Tapping button to launch Safari
info: [debug] Waiting up to 0ms for condition
info: [debug] Pushing command to appium work queue: "au.getElementByAccessibilityId('launch safari')"
info: [debug] Sending command to instruments: au.getElementByAccessibilityId('launch safari')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৩ +0000 Debug: Got new command 2 from instruments: au.getElementByAccessibilityId('launch safari')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৩ +0000 Debug: evaluating au.getElementByAccessibilityId('launch safari')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৩ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৩ +0000 Debug: Lookup returned [object UIAButton] with the name "launch safari" (id: 0).
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৩ +0000 Debug: responding with:
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৩ +0000 Debug: Running system command #3: /usr/local/bin/node **<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":{"ELEMENT":"0"}}...
info: [debug] Socket data received (38 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":{"ELEMENT":"0"}}
info: [debug] Pushing command to appium work queue: "au.tapById('0')"
info: [debug] Sending command to instruments: au.tapById('0')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৪ +0000 Debug: Got new command 3 from instruments: au.tapById('0')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৪ +0000 Debug: evaluating au.tapById('0')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৪ +0000 Debug: UIAButton.tap()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৪ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৪ +0000 Debug: responding with:
info: [debug] Socket data received (25 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":""}
info: [debug] Clicked button, safari should be launching.
info: [debug] Waiting for 6000 ms before navigating to view.
info: [debug] Navigating to most recently opened webview
info: [debug] [REMOTE] Getting WebKitRemoteDebugger pageArray
info: [debug] Picking webview WEBVIEW_1
info: [debug] Attempting to set context to 'WEBVIEW_1'
info: [debug] [REMOTE] Debugger web socket connected to url [ws://localhost:27753/devtools/page/1]
info: [debug] [REMOTE] Unregistering from page readiness notifications
info: [debug] Waiting for 6000 ms before navigating to view.
info: [debug] Navigating to most recently opened webview
info: [debug] [REMOTE] Getting WebKitRemoteDebugger pageArray
info: [debug] Picking webview WEBVIEW_1
info: [debug] Attempting to set context to 'WEBVIEW_1'
info: [debug] [REMOTE] Disconnecting from remote debugger
info: [debug] [REMOTE] Debugger web socket connected to url [ws://localhost:27753/devtools/page/1]
info: [debug] [REMOTE] Unregistering from page readiness notifications
info: [debug] Device launched! Ready for commands
info: [debug] Setting command timeout to the default of 60 secs
info: [debug] Appium session started with sessionId a094ed38-adc2-4e18-a34d-946c06b53d59
info: <-- POST /wd/hub/session 303 21870.128 ms - 74 
info: --> GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59 {}
info: [debug] Responding to client with success: {"status":0,"value":{"webStorageEnabled":false,"locationContextEnabled":true,"browserName":"Safari","platform":"MAC","javascriptEnabled":true,"databaseEnabled":false,"takesScreenshot":true,"networkConnectionEnabled":false,"warnings":{},"desired":{"browserName":"Safari","platformName":"iOS","platformVersion":"7.0.3","app":"io.appium.SafariLauncher","deviceName":"6ea762d97e84b78f78ba67e2e8c8826ca5f8f865","udid":"6ea762d97e84b78f78ba67e2e8c8826ca5f8f865","automationName":"Appium","name":"./intern_test_codes/ios.mweb.config.js","idle-timeout":60,"mouseEnabled":true,"nativeEvents":true,"locationContextEnabled":true},"platformName":"iOS","platformVersion":"7.0.3","app":"io.appium.SafariLauncher","deviceName":"6ea762d97e84b78f78ba67e2e8c8826ca5f8f865","udid":"6ea762d97e84b78f78ba67e2e8c8826ca5f8f865","automationName":"Appium","name":"./intern_test_codes/ios.mweb.config.js","idle-timeout":60,"mouseEnabled":true,"nativeEvents":true,"safari":true},"sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: <-- GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59 200 6.208 ms - 1003 {"status":0,"value":{"webStorageEnabled":false,"locationContextEnabled":true,"browserName":"Safari","platform":"MAC","javascriptEnabled":true,"databaseEnabled":false,"takesScreenshot":true,"networkConnectionEnabled":false,"warnings":{},"desired":{"browserName":"Safari","platformName":"iOS","platformVersion":"7.0.3","app":"io.appium.SafariLauncher","deviceName":"6ea762d97e84b78f78ba67e2e8c8826ca5f8f865","udid":"6ea762d97e84b78f78ba67e2e8c8826ca5f8f865","automationName":"Appium","name":"./intern_test_codes/ios.mweb.config.js","idle-timeout":60,"mouseEnabled":true,"nativeEvents":true,"locationContextEnabled":true},"platformName":"iOS","platformVersion":"7.0.3","app":"io.appium.SafariLauncher","deviceName":"6ea762d97e84b78f78ba6865","udid":"6ea762d97e84b78f78ba6865","automationName":"Appium","name":"./intern_test_codes/ios.mweb.config.js","idle-timeout":60,"mouseEnabled":true,"nativeEvents":true,"safari":true},"sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: --> POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/url {"url":"about:blank"}
info: [debug] [REMOTE] Navigating to new URL: about:blank
info: [debug] [REMOTE] Remote debugger data sent [{"method":"Page.navigate","params":{"objectGroup":"console","includeCommandLineAPI":true,"doNotPauseOnExceptionsAndMuteConsole":true,"url":"about:blank"},"id":1}...]
info: [debug] [REMOTE] Debugger web socket received data: {"result":{},"id":1}
info: [debug] [REMOTE] Debugger got a message for 'Page.navigate' and have no handler, doing nothing.
info: [debug] [REMOTE] Waiting for frame navigated...
info: [debug] [REMOTE] frame navigated timeout triggered
info: [debug] [REMOTE] Frame navigated, calling cbs
info: [debug] [REMOTE] frame navigated in 0.501 sec.
info: [debug] [REMOTE] Waiting for dom...
info: [debug] [REMOTE] Page loaded, verifying whether ready through readyState
info: [debug] [REMOTE] Checking document readyState
info: [debug] [REMOTE] Sending javascript command
info: [debug] [REMOTE] Remote debugger data sent [{"method":"Runtime.evaluate","params":{"objectGroup":"console","includeCommandLineAPI":true,"doNotPauseOnExceptionsAndMuteConsole":true,"expression":"(function (){return document.readyState;})()","ret...]
info: [debug] [REMOTE] Debugger web socket received data: {"result":{"result":{"type":"string","value":"complete"},"wasThrown":false},"id":2}
info: [debug] [REMOTE] readyState was complete
info: [debug] [REMOTE] Page is ready, calling onload cbs
info: [debug] Responding to client with success: {"status":0,"value":"","sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: <-- POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/url 200 2029.889 ms - 74 {"status":0,"value":"","sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: --> POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/execute_async {"script":"arguments[0](true);","args":[]}
info: [debug] Real device safari test and no custom callback address set, changing callback address to local ip.
info: [debug] Response url for executeAsync is http://192.168.1.145:4723/wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/receive_async_response
info: [debug] [REMOTE] Executing atom in default context
info: [debug] [REMOTE] Sending javascript command
info: [debug] [REMOTE] Remote debugger data sent [{"method":"Runtime.evaluate","params":{"objectGroup":"console","includeCommandLineAPI":true,"doNotPauseOnExceptionsAndMuteConsole":true,"expression":"(function(){return function(){var l=this;\nfunctio...]
info: --> GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/orientation {}
info: [debug] Pushing command to appium work queue: "au.getScreenOrientation()"
info: --> GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/location {}
info: [debug] Responding to client that a method is not implemented
info: <-- GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/location 501 2.425 ms - 158 
info: --> GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/screenshot {}
info: --> POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/touch/longclick {}
info: [debug] Missing params for request: ["x","y"]
info: <-- POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/touch/longclick 400 2.165 ms - 29 
info: [debug] We're in the middle of processing a remote debugger command, waiting to run next command until done
info: [debug] Pushing command to appium work queue: "au.getScreenOrientation()"
info: --> GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/window/current/size {}
info: [debug] [REMOTE] Executing 'get_window_size' atom in default context
info: [debug] [REMOTE] Sending javascript command
info: [debug] [REMOTE] Remote debugger data sent [{"method":"Runtime.evaluate","params":{"objectGroup":"console","includeCommandLineAPI":true,"doNotPauseOnExceptionsAndMuteConsole":true,"expression":"(function(){return function(){var f=this;var aa=wi...]
info: [debug] [REMOTE] Debugger web socket received data: {"result":{"result":{"type":"undefined"},"wasThrown":false},"id":3}
info: [debug] [REMOTE] Debugger web socket received data: {"result":{"result":{"type":"object","value":{"width":320,"height":460}},"wasThrown":false},"id":4}
info: [debug] Responding to client with success: {"status":0,"value":{"width":320,"height":460},"sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: <-- GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/window/current/size 200 31.943 ms - 98 {"status":0,"value":{"width":320,"height":460},"sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: <-- OPTIONS /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/receive_async_response 200 1.046 ms - 2 
info: --> POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/window/current/size {"width":320,"height":460}
info: [debug] Responding to client that a method is not implemented
info: <-- POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/window/current/size 501 1.313 ms - 158 
info: --> POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/receive_async_response {"status":0,"value":true}
info: [debug] Responding to client with success: {"status":0,"value":true,"sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: <-- POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/execute_async 200 103.107 ms - 76 {"status":0,"value":true,"sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: <-- POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/receive_async_response 200 4.122 ms - 2 
info: [debug] Sending command to instruments: au.getScreenOrientation()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২১:৫৪ +0000 Debug: Running system command #4: /usr/local/bin/node /Users/widespacebangladesh/Source_Code/WS_Ad_Automation_Job_Server/jobserver/ws_automation_appium/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":""}...
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:০৯ +0000 Debug: Got new command 4 from instruments: au.getScreenOrientation()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:০৯ +0000 Debug: evaluating au.getScreenOrientation()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:০৯ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:০৯ +0000 Debug: responding with:
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:০৯ +0000 Debug: Running system command #5: /usr/local/bin/node **<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":"UNKNOWN"}...
info: [debug] Socket data received (32 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":"UNKNOWN"}
info: [debug] Setting internal orientation to UNKNOWN
info: [debug] Responding to client with success: {"status":0,"value":"UNKNOWN","sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: [debug] Sending command to instruments: au.getScreenOrientation()
info: <-- GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/orientation 200 764.600 ms - 81 {"status":0,"value":"UNKNOWN","sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১০ +0000 Debug: Got new command 5 from instruments: au.getScreenOrientation()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১০ +0000 Debug: evaluating au.getScreenOrientation()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১০ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১০ +0000 Debug: responding with:
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১০ +0000 Debug: Running system command #6: /usr/local/bin/node **<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":"UNKNOWN"}...
info: [debug] Socket data received (32 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":"UNKNOWN"}
info: [debug] Setting internal orientation to UNKNOWN
info: [debug] Pushing command to appium work queue: "au.capture('screenshotbd168b69-7000-4540-a538-3695ee559a5a')"
info: [debug] Sending command to instruments: au.capture('screenshotbd168b69-7000-4540-a538-3695ee559a5a')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১১ +0000 Debug: Got new command 6 from instruments: au.capture('screenshotbd168b69-7000-4540-a538-3695ee559a5a')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১১ +0000 Debug: evaluating au.capture('screenshotbd168b69-7000-4540-a538-3695ee559a5a')
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১১ +0000 Debug: target.captureRectWithName("{origin:{x:0.00,y:0.00}, size:{height:568.00,width:320.00}}", "screenshotbd168b69-7000-4540-a538-3695ee559a5a")
info: [debug] Socket data received (25 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":""}
info: [debug] Waiting 10000 ms for screenshot to ge generated.
info: [debug] atom did not return yet, checking to see if we are blocked by an alert
info: [debug] Pushing command to appium work queue: "au.alertIsPresent()"
info: [debug] Sending command to instruments: au.alertIsPresent()
info: [debug] atom did not return yet, checking to see if we are blocked by an alert
info: [debug] Pushing command to appium work queue: "au.alertIsPresent()"
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১১ +0000 Screenshot captured.
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১১ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১১ +0000 Debug: responding with:
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১১ +0000 Debug: Running system command #7: /usr/local/bin/node **<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":""}...
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৪ +0000 Debug: Got new command 7 from instruments: au.alertIsPresent()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৪ +0000 Debug: evaluating au.alertIsPresent()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৪ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৪ +0000 Debug: responding with:
info: [debug] Socket data received (28 bytes)
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৪ +0000 Debug: Running system command #8: /usr/local/bin/node **<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":false}...
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":false}
info: [debug] Sending command to instruments: au.alertIsPresent()
info: [debug] atom did not return yet, checking to see if we are blocked by an alert
info: [debug] Pushing command to appium work queue: "au.alertIsPresent()"
info: [debug] Responding to client with success: {"status":0,"value":"iVBORw0KGgoAAAANSUhEUgAAAoAAAARwCAYAAAB6j+UAAAAKQWlDQ1BJQ0MgUHJvZmlsZQAASA2dlndUU9kWh8+9N73QEiIgJfQaegkg0jtIFQRRiUmAUAKGhCZ2RAVGFBEpVmRUwAFHhyJjRRQLg4Ji1wnyEFDGwVFEReXdjGsJ7601896a/cdZ39nnt9fZZ+9917oAUPyCBMJ0WAGANKFYFO7rwVwSE8vE9wIYEAEOWAHA4WZmBEf4RALU/L09mZmoSMaz9u4ugGS72yy/UCZz1v9/kSI3QyQGAApF1TY8fiYX5QKUU7PFGTL/BMr0lSkyhjEyFqEJoqwi48SvbPan5iu7yZiXJuShGlnOGbw0noy7UN6aJeGjjAShXJgl4GejfAdlvVRJmgDl9yjT0/icTAAwFJlfzOcmoWyJMkUUGe6J8gIACJTEObxyDov5OWieAHimZ+SKBIlJYqYR15hp5ejIZvrxs1P5YjErlMNN4Yh4TM/0tAyOMBeAr2+WRQElWW2ZaJHtrRzt7VnW5mj5v9nfHn5T/T3IevtV8Sbsz55BjJ5Z32zsrC+9FgD2JFqbHbO+lVUAtG0GQOXhrE/vIADyBQC03pzzHoZsXpLE4gwnC4vs7GxzAZ9rLivoN/ufgm/Kv4Y595nL7vtWO6YXP4EjSRUzZUXlpqemS0TMzAwOl89k/fcQ/+PAOWnNycMsnJ/AF/GF6FVR6JQJhIlou4U8gViQLmQKhH/V4X8YNicHGX6daxRodV8AfYU5ULhJB8hvPQBDIwMkbj96An3rWxAxCsi+vGitka9zjzJ6/uf6Hwtcim7hTEEiU+b2DI9kciWiLBmj34RswQISkAd0oAo0gS4wAixgDRyAM3AD3iAAhIBIEAOWAy5IAmlABLJBPtgACkEx2AF2g2pwANSBetAEToI2cAZcBFfADXALDIBHQAqGwUswAd6BaQiC8BAVokGqkBakD5lC1hAbWgh5Q0FQOBQDxUOJkBCS...","sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: <-- GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/screenshot 200 6148.441 ms - 47798 {"status":0,"value":"iVBORw0KGgoAAAANSUhEUgAAAoAAAARwCAYAAAB6j+UAAAAKQWlDQ1BJQ0MgUHJvZmlsZQAASA2dlndUU9kWh8+9N73QEiIgJfQaegkg0jtIFQRRiUmAUAKGhCZ2RAVGFBEpVmRUwAFHhyJjRRQLg4Ji1wnyEFDGwVFEReXdjGsJ7601896a/cdZ39nnt9fZZ+9917oAUPyCBMJ0WAGANKFYFO7rwVwSE8vE9wIYEAEOWAHA4WZmBEf4RALU/L09mZmoSMaz9u4ugGS72yy/UCZz1v9/kSI3QyQGAApF1TY8fiYX5QKUU7PFGTL/BMr0lSkyhjEyFqEJoqwi48SvbPan5iu7yZiXJuShGlnOGbw0noy7UN6aJeGjjAShXJgl4GejfAdlvVRJmgDl9yjT0/icTAAwFJlfzOcmoWyJMkUUGe6J8gIACJTEObxyDov5OWieAHimZ+SKBIlJYqYR15hp5ejIZvrxs1P5YjErlMNN4Yh4TM/0tAyOMBeAr2+WRQElWW2ZaJHtrRzt7VnW5mj5v9nfHn5T/T3IevtV8Sbsz55BjJ5Z32zsrC+9FgD2JFqbHbO+lVUAtG0GQOXhrE/vIADyBQC03pzzHoZsXpLE4gwnC4vs7GxzAZ9rLivoN/ufgm/Kv4Y595nL7vtWO6YXP4EjSRUzZUXlpqemS0TMzAwOl89k/fcQ/+PAOWnNycMsnJ/AF/GF6FVR6JQJhIlou4U8gViQLmQKhH/V4X8YNicHGX6daxRodV8AfYU5ULhJB8hvPQBDIwMkbj96An3rWxAxCsi+vGitka9zjzJ6/uf6Hwtcim7hTEEiU+b2DI9kciWiLBmj34RswQISkAd0oAo0gS4wAixgDRyAM3AD3iAAhIBIEAOWAy5IAmlABLJBPtgACkEx2AF2g2pwANSBetAEToI2cAZcBFfADXALDIBHQAqGwUswAd6BaQiC8BAVokGqkBakD5lC1hAbWgh5Q0FQOBQDxUOJkBCS...","sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: --> POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/url {"url":"data:text/html;charset=utf-8,%3C!DOCTYPE%20html%3E%3Ctitle%3Ea%3C%2Ftitle%3E"}
info: [debug] [REMOTE] Navigating to new URL: data:text/html;charset=utf-8,%3C!DOCTYPE%20html%3E%3Ctitle%3Ea%3C%2Ftitle%3E
info: [debug] [REMOTE] Remote debugger data sent [{"method":"Page.navigate","params":{"objectGroup":"console","includeCommandLineAPI":true,"doNotPauseOnExceptionsAndMuteConsole":true,"url":"data:text/html;charset=utf-8,%3C!DOCTYPE%20html%3E%3Ctitle%3...]
info: [debug] [REMOTE] Debugger web socket received data: {"result":{},"id":5}
info: [debug] [REMOTE] Debugger got a message for 'Page.navigate' and have no handler, doing nothing.
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৫ +0000 Debug: Got new command 8 from instruments: au.alertIsPresent()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৫ +0000 Debug: evaluating au.alertIsPresent()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৬ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৬ +0000 Debug: responding with:
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৬ +0000 Debug: Running system command #9: /usr/local/bin/node /Users/widespacebangladesh/Source_Code/WS_Ad_Automation_Job_Server/jobserver/ws_automation_appium/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":false}...
info: [debug] Socket data received (28 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":false}
info: [debug] Sending command to instruments: au.alertIsPresent()
info: [debug] atom did not return yet, checking to see if we are blocked by an alert
info: [debug] Pushing command to appium work queue: "au.alertIsPresent()"
info: [debug] [REMOTE] Waiting for frame navigated...
info: [debug] [REMOTE] frame navigated timeout triggered
info: [debug] [REMOTE] Frame navigated, calling cbs
info: [debug] [REMOTE] frame navigated in 0.501 sec.
info: [debug] [REMOTE] Waiting for dom...
info: [debug] [REMOTE] Page loaded, verifying whether ready through readyState
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৭ +0000 Debug: Got new command 9 from instruments: au.alertIsPresent()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৭ +0000 Debug: evaluating au.alertIsPresent()
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৭ +0000 Debug: evaluation finished
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৭ +0000 Debug: responding with:
info: [debug] [INST] ২০১৬-০২-২৪ ০৭:২২:১৭ +0000 Debug: Running system command #10: /usr/local/bin/node **<MY PROJECT DIR>**/appium/node_modules/appium-uiauto/bin/command-proxy-client.js /tmp/instruments_sock 2,{"status":0,"value":false}...
info: [debug] Socket data received (28 bytes)
info: [debug] Socket data being routed.
info: [debug] Got result from instruments: {"status":0,"value":false}
info: [debug] Sending command to instruments: au.alertIsPresent()
info: [debug] atom did not return yet, checking to see if we are blocked by an alert
info: [debug] Pushing command to appium work queue: "au.alertIsPresent()"
info: [debug] [REMOTE] Checking document readyState
info: [debug] [REMOTE] Sending javascript command
info: [debug] [REMOTE] Remote debugger data sent [{"method":"Runtime.evaluate","params":{"objectGroup":"console","includeCommandLineAPI":true,"doNotPauseOnExceptionsAndMuteConsole":true,"expression":"(function (){return document.readyState;})()","ret...]
info: [debug] [REMOTE] Debugger web socket received data: {"result":{"result":{"type":"string","value":"complete"},"wasThrown":false},"id":6}
info: [debug] [REMOTE] readyState was complete
info: [debug] [REMOTE] Page is ready, calling onload cbs
info: [debug] Responding to client with success: {"status":0,"value":"","sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: <-- POST /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/url 200 2015.738 ms - 74 {"status":0,"value":"","sessionId":"a094ed38-adc2-4e18-a34d-946c06b53d59"}
info: --> GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/title {}
info: <-- GET /wd/hub/session/a094ed38-adc2-4e18-a34d-946c06b53d59/title 500 3.356 ms - 101

**This error loops for 18 times before it gets timeout**

**Comment by scottdixon** [comment]
From memory there were issues with this in earlier versions of 1.4 - can you please try a more recent version such as 1.4.16?

**Comment by skbeacon** [comment]
@scottdixon I couldn't find Appium 1.4.16 for Mac OSX.

**Comment by SrinivasanTarget** [comment]
install via npm install appium

**Comment by scottdixon** [comment]
@skbeacon Appium 1.5 has been released, you'll need to install via NPM: `npm install -g appium`. There's no GUI available for it yet, an issue has been opened: https://github.com/appium/appium-dot-app/issues/532

**Comment by emna24** [comment]
@scottdixon : How can you setup my appium when it is installed by npm?

**Comment by scottdixon** [comment]
@emna24 - can you please clarify: are you having trouble installing or starting Appium?

**Comment by skbeacon** [comment]
@scottdixon Thanks man... Sorry for late reply. I have solved the problem before. my appium directory was having some issues, so pull down that specific version. Anyways good to know your suggestion.

@emna24 For our project we pulled a specific appium version and saved it to our project. This was because our project was not supported by latest ones. We are using the appium from that saved one. In general if you want to install lastest/available appium for npm then you need to do it like: `npm install -g appium`.

**Comment by scottdixon** [comment]
Thanks for the update @skbeacon.

@emna24 if your issue persists please open a separate issue.

**Comment by emna24** [comment]
@scottdixon, @skbeacon : sorry for late reply. I don't open my github always so that i didn't pay attention for your answer. my problem is solved a year ago :D 
Thanks a lot ^^
